### DISCUSSION TITLE
Can't connect to Cloudflare

### DISCUSSION THREAD

**Discussion by Calimerorulez** [koush/scrypted#1371] [Q&A]
Hi,

Just trying to setup aa Cloudflare tunnel, I've followed the instructions, but the console shows and access error:

```
########################
3/10/2024, 10:02:56 AM
########################
starting plugin @scrypted/cloud
node modules /root/.scrypted/volume/plugins/@scrypted/cloud/node18-linux-x64
plugin loaded
starting cloudflared
uncaughtException Error: spawn node_modules/cloudflared/bin/cloudflared EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:284:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn node_modules/cloudflared/bin/cloudflared',
  path: 'node_modules/cloudflared/bin/cloudflared',
  spawnargs: [
    'tunnel',
    'run',
    '--token',
    'xxxxxxxxxxxxxxxxxxxxxxxx-redacted-xxxxxxxxxxxxxxxxxxxxxxx'
  ]
}
```

Copied and pasted the (complete) token, but that doesn't help.

**Comment by koush** [comment]
What installation method is this

**Reply by second2falcon** [reply]
@koush @Calimerorulez little update:

I also used tteck's install script and the Cloud plugin including CloudFlare Tunnel are working for me. 

Side note: I wish I could also use CloudFlare Zero Trust Access on top, but I guess that's a project for another day as currently enabling Zero Trust Access results in the following error:
`error="request filtered by middleware handler (AccessJWTValidator) due to: no access token in request" connIndex=0 event=1 ingressRule=0 originService=https://localhost:39583`

**Comment by Calimerorulez** [comment]
The scrypted cloud plugin on a Debian 12 LXC container under Proxmox (https://tteck.github.io/Proxmox/)

**Reply by koush** [reply]
use the official install script

**Reply by Calimerorulez** [reply]
That fixed it. Thanks :)
